### DISCUSSION TITLE
Served directly from Caddy?

### DISCUSSION THREAD

**Discussion by alainwolf** [tarampampam/error-pages#307] [Q&A] [answered]
Would it be possible to serve these error-page templates from Caddy, without running an additional service.
Not only it wouldn't need an additional server running, but I could use variables from Caddy directly, like  [respond](https://caddyserver.com/docs/caddyfile/directives/respond) `{err.status_code}` `{err.status_text}` but also many others in the "details".

They are both written in Go, and use templating.

Unfortunately I'm myself not fluent in Go or Go templating. But maybe somebody has already done that?

**Comment by tarampampam** [comment, marked as answer]
Might [this doc](https://github.com/tarampampam/error-pages/blob/master/docs/guides/caddy_image.md) be helpful?
